### DISCUSSION TITLE
Feature request duplicate/copy scene

### DISCUSSION THREAD

**Discussion by Erriez** [home-assistant/frontend#8061] [Configuration] [answered]
A copy scene via menu `Configuration | Scenes` would be very helpful to reuse / modify an existing scene under a new name.

A duplicate is available for scripts, but not for scenes:

![HomeAssistantDuplicateScene](https://user-images.githubusercontent.com/8849873/103437216-1c771800-4c25-11eb-9d37-b56e4578ef0a.png)

**Comment by KTibow** [comment]
In the meantime, I believe you could edit scenes.yaml.

**Reply by Erriez** [reply]
Editing `scenes.yaml` is a workaround. I don't understand why Scripts can be easily copied via a `Duplicate` menu, but is missing for `Scenes`.

**Reply by KTibow** [reply]
From what I understand, you couldn't just copy+paste the code. Are you sure this should go in the frontend?

**Reply by Erriez** [reply]
I'd like to see a new menu under Scenes like `Duplicate scene` and `Delete scene`. This is currently only available for Scripts. See the screenshot below:
![image](https://user-images.githubusercontent.com/8849873/104855890-a99dba80-590f-11eb-927a-1ec2db407851.png)

**Comment by frenck** [comment, marked as answer]
Added in https://github.com/home-assistant/frontend/pull/9175
